scripts/portconfig

```diff
@@ -109,12 +109,12 @@ class portconfig(object):
     def set_speed(self, port, speed):
         if self.verbose:
             print("Setting speed %s on port %s" % (speed, port))
-        supported_speeds_str = self.get_supported_speeds(port)
-        if supported_speeds_str:
-            if supported_speeds_str.find(str(speed)) == -1:
-                print('Invalid speed specified: {}'.format(speed))
-                print('Valid speeds:{}'.format(supported_speeds_str))
-                exit(1)
+        supported_speeds_str = self.get_supported_speeds(port) or ''
+        supported_speeds = [int(s) for s in supported_speeds_str.split(',')]
```

Comment:
in case `supported_speeds_str` is blank string (``),  next line will faile, because blank string can't be converted to int, 
```
>>> [int(s) for s in  ''.split(',')]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<stdin>", line 1, in <listcomp>
ValueError: invalid literal for int() with base 10: ''
```

You need to check that value you are casting to int is not blank, 
```
 [int(s) for s in  ''.split(',') if s]
```

Reply:
Thanks for the review. Fixed.